src/Tickets/Commerce/Models/Order_Model.php

```diff
@@ -57,6 +57,15 @@ protected function build_properties( $filter ) {
 			$events_in_order  = (array) Arr::get( $post_meta, [ Order::$events_in_order_meta_key ] );
 			$tickets_in_order = (array) Arr::get( $post_meta, [ Order::$tickets_in_order_meta_key ] );
 
+			$tickets = [];
```

Comment:
We already have `$tickets_in_order`, wouldn't it be redundant to have the tickets in two different places? For performance purposes I'd use our `Lazy` collection or some other way to make it more performant.

@bordoni

Reply:
I saw that `tickets_in_order` was being used elsewhere in the codebase, so I didn't want to break anything by changing the structure.

Reply:
Here's where I saw the reference to `tickets_in_order`: https://github.com/the-events-calendar/event-tickets/blob/3a0958cf333841760ca42feb2f86c3530b9c2929/src/Tickets/Commerce/Repositories/Order_Repository.php#L141-L182

Reply:
The `tickets_in_order` array is specifically an array of ticket IDs that are being stored in the meta as `_tec_tc_order_tickets_in_order`.

Reply:
@Camwyn or @bordoni could y'all offer some feedback regarding the above convo? Do you see a problem in having a `tickets` array alongside the `tickets_in_order` array?

Reply:
I think we should avoid adding all ticket data into the Order Model like above. As this will impact the ORM performance for all other queries even without templates.

1. I would suggest that we add a context variable of the type array of `tickets` and organize the data structure for the child templates as well so that we won't need to include separate classes in the template files like this: https://github.com/the-events-calendar/event-tickets/blob/53233e484be7d86f989c7d0278dd2ea3f0af4298/src/views/v2/emails/template-parts/body/order/ticket-totals/ticket-price.php#L26-L28

2. If we change the order model then we should do it like @juanfra suggested and implement `Lazy_Collection` for the `tickets`